boards/pinetime/doc.txt

```diff
@@ -0,0 +1,12 @@
+/**
+@defgroup    boards_pinetime PineTime
+@ingroup     boards
+@brief       Support for the Pinetime
+
+### General information
+
+"The PINE64 SmartWatch, dubbed "PineTime", is a product of a community effort for an open source smartwatch in collaboration with wearable RTOS and Linux app developers & communities."
+
+See https://wiki.pine64.org/index.php/PineTime#PineTime_DevKit_SWD_Probe for more information.
```

Comment:
Why link a specific section of that page here? Isn't it more safe to link to the top of the page (without `#PineTime_DevKit_SWD_Probe` in the url)

Reply:
Agreed, that was an accident.